package.json

```diff
@@ -2,8 +2,8 @@
   "name": "trusted-types",
   "version": "1.1.1",
   "description": "Polyfill for the Trusted Types",
-  "main": "build/cjs/trustedtypes.api_only.js",
-  "module": "build/esm/trustedtypes.api_only.js",
+  "main": "build/cjs/trustedtypes.full_api_only.js",
```

Comment:
Nit: perhaps `trustedtypes.module.js`, since this is to be used like a ES module?

Reply:
Sure.